### PR TITLE
refactor(Games): Improved code and style for WoF and SM

### PR DESCRIPTION
**Pull request by kyranet**
### Fruity Loops
**Before**:
image
**After**:
image

### Wheel of Fortune
**Before**:
image
**After**:
image

### CODE REVIEW

**Review by PyroTechniac — APPROVED** [review]

File: src/lib/util/Games/Slotmachine.ts [hunk 1]
```diff
@@ -180,29 +177,17 @@ export class Slotmachine {
...
 	}
 
-	/** Dark colour in the attachment */
-	public static DARK_COLOUR = '#202225';
-
-	/** Light colour in the attachment */
-	public static LIGHT_COLOUR = '#FFFFFF';
-
-	/** Success colour in the attachment */
-	public static SUCCESS_COLOUR = '#00C853';
```
Comment (cfanoulis) [inline src/lib/util/Games/Slotmachine.ts]: used by `higherlower` as well, please edit it as well

File: src/lib/util/Games/Slotmachine.ts [hunk 2]
```diff
@@ -180,29 +177,17 @@ export class Slotmachine {
...
-
-	/** Failure colour in the attachment */
-	public static FAIL_COLOUR = '#C62828';
```
Comment (cfanoulis) [inline src/lib/util/Games/Slotmachine.ts]: used by `higherlower` as well, please edit it as well

**Review by cfanoulis — CHANGES_REQUESTED** [review]
~~some stuff we shall not forget~~ \\/

**Review by cfanoulis — APPROVED** [review]
neeeeeeeeeeeeeeeevermind

**Pull request merged**